<launch>
  <arg name="dynamic" default="dynamic"/>
  <node pkg="move_base_fork" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/>
    <param name="controller_frequency" value="10"/>
    <rosparam file="$(find crowdbot_active_slam)/nav/pepper_realsense_costmap_common_params_$(arg dynamic).yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find crowdbot_active_slam)/nav/pepper_realsense_costmap_common_params_$(arg dynamic).yaml" command="load" ns="local_costmap"/>
    <rosparam file="$(find crowdbot_active_slam)/nav/pepper_local_costmap_params.yaml" command="load"/>
    <rosparam file="$(find crowdbot_active_slam)/nav/pepper_global_costmap_params.yaml" command="load"/>
    <rosparam file="$(find crowdbot_active_slam)/nav/pepper_base_local_planner_params.yaml" command="load"/>
    <remap from="map" to="occupancy_map"/>
    <remap from="odom" to="pepper_robot/odom"/>
    <remap from="scan" to="combined_scan_sync"/>
    <remap from="move_base/TebLocalPlannerROS/obstacles" to="static_scan_extractor/dyn_obstacles"/>
  </node>
</launch>
